File: py files/4c.py
# 4c: Create an array of numbers in Python and write a function to find the highest number in the array.

def find_highest_number(numbers):
    """Returns the highest number in a list"""
    if not numbers:
        return None
    
    highest = numbers[0]
    for num in numbers:
        if num > highest:
            highest = num
    
    return highest

if __name__ == "__main__":
    number_array = [23, 67, 12, 89, 45, 34, 91, 56]
    
    print(f"Numbers: {number_array}")
    highest = find_highest_number(number_array)
    print(f"The highest number is: {highest}")

File: js files/4c.js
// 4c. Create an array of numbers in JS and write a function to find the highest number in the array.

function findHighestNumber(numbers) {
    if (numbers.length === 0) {
        return null;
    }
    
    let highest = numbers[0];
    for (let num of numbers) {
        if (num > highest) {
            highest = num;
        }
    }
    
    return highest;
}

const numberArray = [23, 67, 12, 89, 45, 34, 91, 56];

console.log(`Numbers: ${numberArray}`);
const highest = findHighestNumber(numberArray);
console.log(`The highest number is: ${highest}`);